### ISSUE TITLE
Resize Windowss manually 

### ISSUE THREAD

**Issue by ccoupe** [shoes/shoes3#430]
Shoes 3.3.7 for Linux and Windows has a bug when manually resizing windows that have a menubar - they don't shrink.

**Comment by tuttza** [comment]
@ccoupe I'll take a look at the scroll bar issue.

**Comment by ccoupe** [comment]
There is a issue talking about that problem. I can't seem to find it, now. Good Luck. Fixing one has the effect of regressing the other.

**Comment by ccoupe** [comment]
One thing we can do is to use the GTK type checking macros for calls to Gtk functions. The macros can/do expand to run time type checking and I've played fast and loose with casting. That could be the problem. And it's just good practice.

**Comment by tuttza** [comment]
Yeah I also think that might be causing some issues, also some warnings about it as well when compiling. I'm working on fixing these, hopefully really fix the resize issue.
